doc/Introduction.html

```diff
@@ -69,30 +69,31 @@ <h2 class="sidebar-projectVersion">
       
         <section id="moduledoc" class="docstring">
           <h1>Welcome to the Airport</h1>
-<p>You made it ! Let’s go through some information before you get on the plane.</p>
+<p>You made it! Let’s go through some information before you get on the plane.</p>
 <p>On each destination you’ll receive information of your next trip.</p>
 <p>Here are suggestions for your safe and successful travels.</p>
 <ul>
-<li>Remember that you are an engineer and you are here to create solutions.
+<li>Remember that you are a great student and you are here to create solutions.
```

Comment:
@alan-andrade what do you think about using "students" instead of "engineers"? I think this way we can make people feel more identified and comfortable with their current role in the workshop. :smile_cat: